test/libsolidity/gasTests/abiv2_optimised.sol

```diff
@@ -17,9 +17,9 @@ contract C {
 // optimize-yul: true
 // ----
 // creation:
-//   codeDepositCost: 603000
-//   executionCost: 638
-//   totalCost: 603638
+//   codeDepositCost: 610600
+//   executionCost: 645
+//   totalCost: 611245
```

Comment:
This *increasing* is weird, though, that I don't quite understand yet...

Reply:
Even weirder: if I generally add ``_ptr`` to ``identifierLocationSuffix`` for memory references, then this seems to decrease again...

Reply:
I'll try to understand this by looking at the generated code...

Reply:
Ok, this is weird... https://gist.github.com/ekpyron/8639d242e9a331bcefa8d02ed185f133
Special attention to the validator functions at the very end of the gist and their uses (or lack thereof)...

Reply:
Ah ok... I think it's just inlining that is different... probably due to something being sorted by identifier somewhere...